docs/components/headings/README.md

```diff
@@ -10,11 +10,11 @@
     },
     {
       "text": 'Typography',
-      "href": '/foundation/typography/'
+      "href": '../foundation/typography/'
     },
     {
       "text": 'Paragraphs',
-      "href": '/components/paragraphs/'
+      "href": '../components/paragraphs/'
```

Comment:
as paragraphs is in the same root dir as headings this will need to be ../paragraphs/

(the "../" moves your URL up one directory so this is moving you out of the paragraph dir into the component dir)

Reply:
ok, thanks for the explanation!